pkgs/development/tools/rust/rust-analyzer/default.nix

```diff
@@ -58,13 +69,16 @@ rustPlatform.buildRustPackage rec {
   passthru = {
     updateScript = nix-update-script { };
     # FIXME: Pass overrided `rust-analyzer` once `buildRustPackage` also implements #119942
-    tests.neovim-lsp = callPackage ./test-neovim-lsp.nix { };
+    # tests.neovim-lsp = callPackage ./test-neovim-lsp.nix { };
```

Comment:
Please add a `FIXME` comment with a link to this issue to explain why it is disabled.